packages/web-app-search/src/portals/SearchBar.vue

```diff
@@ -6,6 +6,9 @@
     class="oc-flex"
     data-custom-key-bindings-disabled="true"
   >
+    <span id="search-description" class="oc-invisible-sr">
+      {{ $gettext('Results will appear below the search bar if they exist.') }}
```

Comment:
Since this label is targeting users using screen readers, I wonder whether mentioning "below" would make sense to them. At the same time nothing better comes to mind...